src/tests/JIT/Regression/JitBlue/Runtime_105792/Runtime_105792.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
+  </ItemGroup>
```

Comment:
```suggestion
```
(It won't have any effect without `RequiresProcessIsolation` anyway)

Reply:
I guess we get enough TC=0 coverage with external config settings.